lib/blocks.php

```diff
@@ -328,6 +307,8 @@ static function () {
 	}
 }
 
+// add_filter( 'should_load_separate_core_block_assets', '__return_false', 11 );
+
```

Comment:
```suggestion
```

Looks like this is a left over from testing/debugging.

Reply:
Ah yes that should be removed 😅 thanks!

Reply:
Saved me some typing while testing though 😁